contracts/ExampleToken-v2.cdc

```diff
@@ -18,6 +18,7 @@ access(all) contract ExampleToken: ViewResolver {
     /// User Paths
     access(all) let VaultStoragePath: StoragePath
     access(all) let VaultPublicPath: PublicPath
+    access(all) let ReceiverPublicPath: PublicPath
```

Comment:
Why add a receiver public path? We already have the vault public path. With entitlements, there is no difference any more

Reply:
In the context of this single contract, I agree - may as well just use a single storage & public path.

But when incorporating `TokenForwarding`, I ran into an issue wiring up the' `Receiver` implementing `Forwarder` resource at the `ExampleToken.VaultPublicPath`. `Forwarder` doesn't conform to the `Vault` interface so it can only be linked as a `Receiver` and doing so at the `.VaultPublicPath` breaks expectations for getters trying to retrieve `&{FungibleToken.Vault}`.

I thought creating another canonical path dedicated to the `Receiver` functionality would allow for more granularity in cases like these.

Does that make sense?

Reply:
that is a good point. Might need to consider adding that to the standard itself and adding a comment explaining why we still need both paths. I guess it is kind of similar to how we have the receiver public path and the balance public path as separate paths in the v1 standard. Can you make that change in the standard?

Reply:
I don't think we can add those paths as contract fields in the v2 standard. FungibleToken will no longer be a contract interface, so those values would need to be initialized and lead to collisions between vaults, wouldn't they?

Reply:
thank you!